distribution/kernel/carbon-home/repository/conf/carbon.xml

```diff
@@ -491,6 +523,7 @@
 	<!--Option to disable storing of tokens issued by STS-->
 	<DisableTokenStore>false</DisableTokenStore>
 
+        <STSCallBackHandlerName>org.wso2.carbon.identity.sts.common.identity.provider.AttributeCallbackHandler</STSCallBackHandlerName>
```

Comment:
Kernel will not have this class by default. What would be the behavior in that case?

Reply:
Was not aware of it. It's there in the default.json of the kernel 
https://github.com/wso2/carbon-kernel/blob/4.6.x/distribution/kernel/carbon-home/repository/resources/conf/default.json#L188

Reply:
This has been added by https://github.com/wso2/carbon-kernel/pull/2685 with the changes done sts. When starting up the kernel and accessing it I did not see any errors as well.